### PR TITLE
New package: streamway.obs-plugin version 1

### PR DESCRIPTION
**Pull request by Ramjivan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138354)

### CODE REVIEW

**Comment by Ramjivan** [comment]
> @Ramjivan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Shri Streamway Tech Private Limited"]

**Comment by Ramjivan** [comment]
@microsoft-github-policy-service agree [company="Shri Streamway Tech Private Limited"]

**Comment by Ramjivan** [comment]
@microsoft-github-policy-service agree company="Shri Streamway Tech Private Limited"

**Comment by stephengillie** [comment]
Hi @Ramjivan,

Should `OBSProject.OBSStudio` be a dependency?

**Comment by Ramjivan** [comment]
Hi @stephengillie 
Yes OBSProject.OBSStudio is required dependency. I couldn't find any way how to declare that in manifest, can you point me to exact doc? or should I implement it in installer itself to check if OBS is installed or not. ?

File: manifests/s/streamway/obs-plugin/1/Streamway.OBSPlugin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Streamway.OBSPlugin
+PackageVersion: "1"
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/s/streamway/obs-plugin/1/Streamway.OBSPlugin.installer.yaml:6]:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
    - PackageIdentifier: OBSProject.OBSStudio
```

**Comment by stephengillie** [comment]
> Hi @stephengillie Yes `OBSProject.OBSStudio` is required dependency. I couldn't find any way how to declare that in manifest, can you point me to exact doc? or should I implement it in installer itself to check if OBS is installed or not. ?

Here's a link to our 1.6.0 manifest schema: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/installer.md

File: manifests/s/streamway/obs-plugin/1/Streamway.OBSPlugin.installer.yaml (hunk 1 above)
Comment (Ramjivan) [reply manifests/s/streamway/obs-plugin/1/Streamway.OBSPlugin.installer.yaml:6]: Thanks for suggestion, Committing it.

**Comment by Ramjivan** [comment]
Hi @stephengillie  
Since i have changed YAML file names, do i need to close this and start a new PR ?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-13T04:12:19.3594957Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\streamway\obs-plugin\1', expected 'manifests\s\Streamway\OBSPlugin\1') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\streamway\obs-plugin\1', expected 'manifests\s\Streamway\OBSPlugin\1'

(Automated response - build 729.)

**Comment by Ramjivan** [comment]
Hi @stephengillie 

Thank You so much for all the help. 😊

**Comment by stephengillie** [comment]
Verify install: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**